modelcheck/gradle/dependency-locks/compileClasspath.lockfile

```diff
@@ -1,17 +1,22 @@
 # This is a Gradle generated file for dependency locking.
 # Manual edits can break the build and are not advised.
 # This file is expected to be part of source control.
-com.fasterxml.jackson.core:jackson-annotations:2.11.2
-com.fasterxml.jackson.core:jackson-core:2.11.2
-com.fasterxml.jackson.core:jackson-databind:2.11.2
-com.fasterxml.jackson.dataformat:jackson-dataformat-xml:2.11.2
-com.fasterxml.jackson.module:jackson-module-jaxb-annotations:2.11.2
+com.fasterxml.jackson.core:jackson-annotations:2.12.0-rc1
```

Comment:
Please double check with master if these versions are compatible. We had problems with fasterxml because MPS ships a version as well and we needed to make sure the are compatible.